modules/solid_properties/doc/content/source/materials/TungstenThermalPropertiesMaterial.md

```diff
@@ -0,0 +1,17 @@
+# Tungsten Thermal Properties Material
+
+!syntax description /Materials/TungstenThermalPropertiesMaterial
+
+`TungstenThermalPropertiesMaterial` provides tungsten's thermal conductivity, specific heat, and density as material properties based on temperature
+
+Properties of tungsten are obtained from [!cite](milner2024space).
```

Comment:
Please provide the expression for each property, along with the units and the temperature range of validity.